### DISCUSSION TITLE
Why is ``Mark as read/unread`` hidden behind a dropdown?

### DISCUSSION THREAD

**Discussion by marwie** [community/community#33800] [Other Feature Feedback, Questions, & Ideas]
Hello,

I'm wondering why options like ``Mark as read`` are hidden behind the dropdown button altough there would be enough space to the right to just display them there. With the dropdown it's two clicks while also making it harder to discover that functionality.

![image](https://user-images.githubusercontent.com/5083203/191690029-8d2b6ffc-b70d-4207-b4e9-70d82c993c78.png)

**Comment by jfullstackdev** [comment]
maybe because the developers preferred clicking the button `Done` to encourage opening those notifs first than simply marking it as `Read`, and in that way, we can also make sure we'll not do as much `Unread`
